versatileimagefield/templates/django/forms/widgets/attrs.html

```diff
@@ -0,0 +1 @@
+{% for name, value in widget.attrs.items %}{% if value != False %} {{ name }}{% if value != True %}="{{ value }}"{% endif %}{% endif %}{% endfor %}
```

Comment:
Aah I see. I think it would be helpful to add this explanation to the template, just to be safe. Or maybe even move the template into the versatileimagefield templates folder, since this is only relevant on  older Django versions? It's not my project, but think it would be cleaner to not override Django's own templates if it's not necessary.